### PR TITLE
Added CODEX Dockerfile, R script, and python script for VCF conversion.

### LINKED ISSUE #7: Build docker and write WDL for running CODEX pipeline

### PR DESCRIPTION
**Pull request by samuelklee**
@asmirnov239 @mbabadi I will add the WDL in a later commit to this PR after I finish it and test it out.  But can you quickly look at the changes I made to the VCF header and conversion script?  I removed many annotations that seemed extraneous.  Also, I think the original calculations for NT and AN might have been incorrect.  Finally, it seems like we use CODEX's `lratio` as GQ, but I think this is simply the log ratio of the normalized vs. raw coverage?

Closes #7.

### CODE REVIEW

**Comment by samuelklee** [comment]
@asmirnov239 @mbabadi TCGA and GPC2 runs are done, so this PR can probably go in.  If you want me to make any changes to the VCF output, the WDL is set up so that we can rerun using cached CODEX results.

File: CNV/gCNV/codex/resources/codex_vcf_converter.py [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+              + 'NT=' + str(num_targets)
+
+    return info_field
+
+
+def _construct_genotype_field(
+        call_fields: Dict[str, str]) -> str:
+    # parse call fields
+    cnv = call_fields['cnv']
+    lratio = int(float(call_fields['lratio']))
```
Comment (asmirnov239) [inline CNV/gCNV/codex/resources/codex_vcf_converter.py]: We had some troubles interpreting the results of the seg file in the gCNV Spark evaluations. Here what their documentation says: lratio (likelihood ratio of CNV event versus copy neutral event). So it should be a proxy for GQ, however it yields strange results..

File: CNV/gCNV/codex/resources/codex_vcf_converter.py [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+    # construct genotype field
+    if cnv == 'del':
+        gt = '1'
+        pl = str(lratio) + ',0,10000'
+    elif cnv == 'dup':
+        gt = '2'
+        pl = str(lratio) + ',10000,0'
+    else:
+        raise Exception('CODEX \"cnv\" field must be \"del\" or \"dup\".')
+    gq = min(99, lratio)
```
Comment (asmirnov239) [inline CNV/gCNV/codex/resources/codex_vcf_converter.py]: Do we have to cap the GQ at 99?

Comment (samuelklee) [reply CNV/gCNV/codex/resources/codex_vcf_converter.py]: I guess not...this was just a holdover from the script you gave me.  I can change it.

File: CNV/gCNV/codex/docker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,6 @@
+FROM bioconductor/release_base2
```
[the remark this answers is not in the document]
Comment (samuelklee) [reply CNV/gCNV/codex/docker/Dockerfile]: Yup!

**Pull request merged**